sdk/python/tests/integration/feature_repos/universal/data_sources/file.py

```diff
@@ -46,3 +49,79 @@ def create_offline_store_config(self) -> FeastConfigBaseModel:
 
     def teardown(self):
         self.f.close()
+
+
+class S3FileDataSourceCreator(DataSourceCreator):
+    f: Any
+    minio: DockerContainer
```

Comment:
We should add a section in our developer guide that running this test (and potentially others in the future) locally requires docker-desktop installed.

Reply:
Added.